tests/core/Blockchain.spec.ts

```diff
@@ -10,6 +10,33 @@ import TransactionModel from '../../lib/common/models/TransactionModel';
 import ValueTimeLockModel from '../../lib/common/models/ValueTimeLockModel';
 
 describe('Blockchain', async () => {
+  describe('startPeriodicCachedBlockchainTimeRefresh', () => {
+    beforeEach(() => {
+      // Freeze time
+      jasmine.clock().install();
```

Comment:
Sanity: what's this jasmine.clock().install(); witchcraft? Was it left over during debug or something?